ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>We don't have a way to gather results from CLI right now. Maybe it's worth saving the Python API results to disk and then here we re-load them? Not sure if that's too complicated for users though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>